### DISCUSSION TITLE
integrazione HA tapo p100

### DISCUSSION THREAD

**Discussion by federico89mi** [indomus/forum#1157] [HUB: Home Assistant]
Buongiorno,
vi vorrei chiedere un aiuto riguardo un problema che ho in riferimento a questo articolo: https://indomus.it/componenti/tp-link-tapo-p100/.
In poche parole io installo tramite hacs l'integrazione, riavvio HA però poi non trovo nulla con nome tapo nelle integrazioni ufficiali di home assistant...

Mi sapete dire il motivo?

**Comment by federico89mi** [comment]
nessuno sa aiutarmi? Posso dirvi che non ho fato altro che installare da Hacs l'integrazione e riavviato ma poi non appare nelle integrazioni di HA

**Comment by petretiandrea** [comment]
Ciao Federico, non ci sono ancora integrazioni "ufficiali", a ogni modo io ne ho realizzata una https://github.com/petretiandrea/home-assistant-tapo-p100. È installabile attraverso uno store chiamato HACS, per cui prima dovresti installare HACS su home assistant trovi varie guide online e anche su indomus se non sbaglio. Una volta fatto, basta che cerchi "tapo" tra le integrazioni offerte da HACS, lo installi e riavvii home assistant a quel punto troverai l'integrazioni tra quelle utilizzabili di home assistant.
